utils/cache.js

```diff
@@ -59,25 +69,64 @@ const cachePool = (opt = { maximumSize: CACHE_SIZE_MB }) => {
     }
   };
 
-  return { get, set, hits, cacheStore };
+  return { get, set, evict, hits, cacheStore };
+};
+
+const cachedKeysStore = () => {
```

Comment:
here we are not checking if the modelKey property already exists in the keyStore object. This means that if the modelKey property already exists in the keyStore object, the function will add the cachedKey property to the keyStore object twice.

Reply:
This is the function where I'm adding a cachedKey. Here, I am checking if there is a model key present and adding a cachedKey based on that. Please let me know if I got it wrong.


```javascript
 const addCachedKey = (modelKey, cachedKey) => {
     if (keyStore.has(modelKey)) {
       keyStore.get(modelKey).add(cachedKey);
     } else {
       const set = new Set();
       set.add(cachedKey);
       keyStore.set(modelKey, set);
     }
   };
``` 
PR link link: https://github.com/Real-Dev-Squad/website-backend/pull/1283/files#diff-5ceb95acc7e3f0b7328eba1584f1eb987b65f01cb9c396d74a733d3a6b0565b9R86